pom.xml

```diff
@@ -36,6 +36,30 @@
 			<artifactId>spring-boot-starter-data-mongodb</artifactId>
 		</dependency>
 		
+		<dependency>
+			<groupId>de.flapdoodle.embed</groupId>
+			<artifactId>de.flapdoodle.embed.mongo</artifactId>
+			<scope>test</scope>
+		</dependency>
+		
+		<dependency>
+			<groupId>net.rakugakibox.spring.boot</groupId>
+			<artifactId>orika-spring-boot-starter</artifactId>
+			<version>1.9.0</version>
+		</dependency>
```

Comment:
Why are you adding this dependency?

Reply:
For convert the object model to the domain

Reply:
This dependency is used by the StoryMapping class that you're not using, please delete it.